### PR TITLE
snowflake-arctic-base

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
Added Model snowflake-snowflake-arctic-base

### CODE REVIEW

File: assets/models/system/snowflake-snowflake-arctic-base/description.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Inference samples
+Inference type|Python sample (Notebook)|CLI with YAML
+|--|--|--|
+Real time|<a href="https://aka.ms/azureml-infer-online-sdk-text-generation-dolly" target="_blank">text-generation-online-endpoint.ipynb</a>|<a href="https://aka.ms/azureml-infer-online-cli-text-generation-dolly" target="_blank">text-generation-online-endpoint.sh</a>
+Batch |<a href="https://aka.ms/azureml-infer-batch-sdk-text-generation" target="_blank">text-generation-batch-endpoint.ipynb</a>| coming soon
+
+
+# Sample Inputs and Outputs (for real-time inference)
+### Sample Input
```
Comment (msp8955) [inline assets/models/system/snowflake-snowflake-arctic-base/description.md:32]: Can we add sample (please follow llama-2-7b sample formatting which is also text-generation model)
Sample input
{"input_data": {"input_string": [ "I believe the meaning of life is" ], "parameters": {"max_tokens": 200, "return_full_text": false, "do_sample": true, "top_p": 0.9, "temperature": 0.6 } }}

Sample output
[{"0":"  The meaning of life is a deeply personal and philosophical question that has been debated throughout human history. Different people and cultures have their own interpretations, often influenced by religion, spirituality, and personal experiences. Some common themes include seeking happiness, fulfillment, and connection with others, as well as personal growth and self-discovery. Ultimately, the meaning of life is subjective and can be unique to each individual."}]

File: assets/models/system/snowflake-snowflake-arctic-base/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: snowflake-snowflake-arctic-base
```
Comment (msp8955) [inline assets/models/system/snowflake-snowflake-arctic-base/spec.yaml:2]: why are these folder names and model names have double "snowflake"?
should be snowflake-arctic-base

File: assets/models/system/snowflake-snowflake-arctic-base/description.md (hunk 1 above)
Comment (KonakanchiSwathi) [reply assets/models/system/snowflake-snowflake-arctic-base/description.md]: Added the sample input and output as per llama-2-7b

File: assets/models/system/snowflake-snowflake-arctic-base/spec.yaml (hunk 2 above)
Comment (KonakanchiSwathi) [reply assets/models/system/snowflake-snowflake-arctic-base/spec.yaml]: Chnaged the foldername and model name

**Review by sanchez-alex — APPROVED** [review]

**Review by skasturi — APPROVED** [review]

**Pull request merged**